Content.Shared/Roles/JobRequirements.cs

```diff
@@ -7,24 +7,29 @@
 namespace Content.Shared.Roles
 {
     /// <summary>
-    ///     Abstract class for playtime and other requirements for role gates.
+    /// Abstract class for playtime and other requirements for role gates.
     /// </summary>
     [ImplicitDataDefinitionForInheritors]
-    public abstract class JobRequirement
-    {
-        [DataField("inverted")] public bool Inverted;
-    }
+    public abstract class JobRequirement{}
 
     [UsedImplicitly]
     public sealed class DepartmentTimeRequirement : JobRequirement
     {
+        /// <summary>
+        /// Which department needs the required amount of time
+        /// </summary>
         [DataField("department", customTypeSerializer: typeof(PrototypeIdSerializer<DepartmentPrototype>))]
         public string Department = default!;
 
         /// <summary>
         /// How long (in seconds) this requirement is.
         /// </summary>
         [DataField("time")] public TimeSpan Time;
+
+        /// <summary>
+        /// Should the condition be inverted?.
```

Comment:
Be more concrete than this, declare what is the purpose of the condition being inverted